docs/genomic_file/README.md

```diff
@@ -0,0 +1,85 @@
+# Model Updates
+
+## CavaticaTask Generalization
+Up until this point we have only ingested harmonized files from Cavatica. Now we will also receive
+harmonized files from the Broad Institute.
+
+### Implementation
+We need to rename the models and fields to generalize the CavaticaTask and CavaticaTaskGenomicFile models.
+
+<p align="center">
+  <img src="cav-task-model-update.png">
+</p>
+
+### Migration
+Migration should be fairly simple. It will consist of alter columns and alter table commands.
+Also, we don't have any data in cavatica_task, cavatica_task_genomic_files tables.
+
+## Multi-specimen Genomic Files
+We decided that we will not have a separate model to represent multi-specimen genomic files.
+We will keep the many-to-many relationship between biospecimens and genomic files.
+
+## Sequencing Experiments and Harmonized Files
+
+People need to be able to search for files by various sequencing experiment attributes.
+
+### Model Change
+
+1. Add the common SE fields to GenomicFile. Do not remove them from
+SequencingExperiment yet (we want to keep them for now)
+
+    - `experiment_strategy` (text or array)
+    - `platform` (text or array)
+    - `instrument_model` (text or array)
+    - `is_paired_end` (boolean or array)
+
+\* At the moment it's unlikely for these to have different values,
+but as with many things bioinformatics nothing is impossible. Initially, we
+should have a standard enum to indicate that there are multiple values. However,
+it would be more ideal to store the values as arrays. We need to explore the
```

Comment:
What is the motivation of using arrays? If we want to use arrays, it would make more sense to have these fields exist in a different model with one-many relationship.

Reply:
1) Really the "raw data" from a sequencing experiment is the read-level data, that is FASTQ, BAM, CRAMs - so they make the most sense to be directly tied to a sequencing experiment 

2) Everything else, gVCF, VCF, etc is a processing level that is using that raw data to determine genomic variants/features, but they don't contain the reads themselves 

3) However, when people are looking for variant data they typically want to search and filter by whether those gVCF/VCF came from whole genome vs whole exome, and now with newer platforms/instruments like pacbio/nanopore appearing it's likely also want to distinguish between Illumina and those

4) So to meet the searchability need with the current model we could either attach sequencing experiment directly to those files, resulting in many-to-many relationships, require the portal ETL to walk through the logic, or push the fields needed for searching onto the files themselves

5) After some discussion, the last one was chosen. Traditionally things like gVCF/VCFs were typically only created with one type of experimental strategy, but increasingly there have been new methods that include multiple strategies to improve power. For example, high depth whole exome sequencing combined with whole genome for wider coverage (this is actually how the Kids First sequencing is done for tumors) as well as combining RNA-seq with DNA methods as they're all different inspections of the same underlying information. 

6) To address that potential if it was easy to utilize arrays that would help prevent unnecessary changes in the future. However, sounds like there is both some technical challenges as well as the consideration that we might want the search fields to say something like "WXS/WGS" versus bucketing them into the two categories. 

7) So if it's easier to get this PR through - it should be fine to go ahead and use it as a text field, just wanted to capture the above